linker/shared/src/main/scala/org/scalajs/linker/checker/IRChecker.scala

```diff
@@ -1327,14 +1333,18 @@ private final class IRChecker(unit: LinkingUnit, logger: Logger) {
           classDef.ancestors.toSet,
           classDef.hasInstances,
           classDef.jsNativeLoadSpec,
-          CheckedClass.checkedFieldsOf(classDef))
+          CheckedClass.checkedFieldsOf(classDef),
+          classDef.jsNativeMembers.map(_.name.name).toSet)
```

Comment:
Is it intended that this basically allows `JSNativeMemberDefs` on any type of class?